src/loki/values/values.yaml

```diff
@@ -25,6 +25,15 @@ loki:
       insecure: false
   commonConfig:
     replication_factor: 1
+  schemaConfig:
+    configs:
+      - from: 2024-04-01
```

Comment:
I think this should be set to a date in the future based on https://grafana.com/docs/loki/latest/operations/storage/schema/#changing-the-schema, otherwise it will result in logs from 4/1 -> today being unreadable since they are actually v12 schema? I think we could set it to one month+ out explicitly, OR we could look at doing this via zarf var/pepr to mutate in a date in the future.

I'm not certain but we may also need a v12 schema entry here to handle older logs (using a `from` date of something last year to safely handle all possible logs from the first uds-core release)?